go.mod

```diff
@@ -9,11 +9,11 @@ require (
 	github.com/kr/pretty v0.2.1
 	github.com/peterhellberg/link v1.1.0
 	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127
-	gopkg.in/yaml.v2 v2.4.0
+	gopkg.in/yaml.v2 v2.2.2
```

Comment:
This requirement can be removed, and replaced everywhere by `sigs.k8s.io/yaml`

Reply:
Done, thanks for pointing out